llvm/include/llvm/SYCLLowerIR/ESIMD/ESIMDUtils.h

```diff
@@ -68,5 +74,35 @@ void collectUsesLookThroughCastsAndZeroGEPs(const Value *V,
 /// in it. Returns nullptr if failed to do so.
 Type *getVectorTyOrNull(StructType *STy);
 
+// Simplest possible implementation of an allocator for the Itanium demangler
```

Comment:
I think this change is no longer needed

Reply:
Upon second thought, it is good to have the allocator defined in shared header, as others may need to use it too.